Admin.js

```diff
@@ -114,561 +141,775 @@ var Admin = {
+                insertBefore(jQuery(this).parent()).
+                trigger("sonata-admin-append-form-element");
+            jQuery(this).trigger("sonata-collection-item-added");
+
         });
 
-        jQuery(subject).on('click', '.sonata-collection-delete', function(event) {
+        jQuery(subject).on("click", ".sonata-collection-delete", function (event) {
+
             Admin.stopEvent(event);
 
-            jQuery(this).trigger('sonata-collection-item-deleted');
+            jQuery(this).trigger("sonata-collection-item-deleted");
 
-            jQuery(this).closest('.sonata-collection-row').remove();
+            jQuery(this).closest(".sonata-collection-row").
+                remove();
+
+            jQuery(document).trigger("sonata-collection-item-deleted-successful");
 
-            jQuery(document).trigger('sonata-collection-item-deleted-successful');
         });
+
     },
 
-    setup_per_page_switcher: function(subject) {
-        Admin.log('[core|setup_per_page_switcher] setup page switcher', subject);
+    "setup_per_page_switcher" (subject) {
+
+        Admin.log("[core|setup_per_page_switcher] setup page switcher", subject);
+
+        jQuery("select.per-page").change(function (event) {
 
-        jQuery('select.per-page').change(function(event) {
-            jQuery('input[type=submit]').hide();
+            jQuery("input[type=submit]").hide();
+
+            window.top.location.href = this.options[this.selectedIndex].value;
 
-            window.top.location.href=this.options[this.selectedIndex].value;
         });
+
     },
 
-    setup_form_tabs_for_errors: function(subject) {
-        Admin.log('[core|setup_form_tabs_for_errors] setup form tab\'s errors', subject);
+    "setup_form_tabs_for_errors" (subject) {
+
+        Admin.log("[core|setup_form_tabs_for_errors] setup form tab's errors", subject);
 
         // Switch to first tab with server side validation errors on page load
-        jQuery('form', subject).each(function() {
-            Admin.show_form_first_tab_with_errors(jQuery(this), '.sonata-ba-field-error');
+        jQuery("form", subject).each(function () {
+
+            Admin.show_form_first_tab_with_errors(jQuery(this), ".sonata-ba-field-error");
+
         });
 
         // Switch to first tab with HTML5 errors on form submit
-        jQuery(subject)
-            .on('click', 'form [type="submit"]', function() {
-                Admin.show_form_first_tab_with_errors(jQuery(this).closest('form'), ':invalid');
-            })
-            .on('keypress', 'form [type="text"]', function(e) {
-                if (13 === e.which) {
-                    Admin.show_form_first_tab_with_errors(jQuery(this), ':invalid');
+        jQuery(subject).
+            on("click", "form [type=\"submit\"]", function () {
+
+                Admin.show_form_first_tab_with_errors(jQuery(this).closest("form"), ":invalid");
+
+            }).
+            on("keypress", "form [type=\"text\"]", function (e) {
```

Comment:
- eslint: Unexpected unnamed function.
- eslint: Identifier name 'e' is too short (< 2).